tests/test_name_version.py

```diff
@@ -12,4 +12,4 @@ def test_get_app_and_version(backend, backend_name):
     app_name, version = unpack_get_app_and_version_response(response.data)
 
     assert app_name == "TON"
-    assert version == "2.3.0"
+    assert version == "2.4.0"
```

Comment:
Could be interesting to find the value in the makefile directly, instead of hadcode the value here. See an example here: https://github.com/LedgerHQ/app-xrp/blob/cc94b699f837015abb9423ac6afeca7be1972fff/tests/utils.py#L57